components/alert.css

```diff
@@ -0,0 +1,43 @@
+.display-alert{
+    display: flex;
+    align-items: center;
+    margin-top: 1rem;
+}
+
+.alert-item{
+    display: flex;
+    align-items: center;
+    padding: 1rem;
+    font-weight: 400;
+    border-radius: 4px;
+    min-width: 50%;
+}
+
+.alert-icon{
+    margin-right: 1rem;
+    font-size: 1.5rem;
+}
+
+.alert-success{
+    background-color: var(--light-green);
```

Comment:
it's better to name color variables as the purpose instead of the color itself. For example "success-color"

Reply:
Sure, will make the changes

Reply:
Made the changes. Thanks!